vaadin-spring-tests/pom.xml

```diff
@@ -137,7 +137,51 @@
                 <artifactId>vaadin-text-field-testbench</artifactId>
                 <version>${vaadin.version}</version>
             </dependency>
-
+            <dependency>
+                <groupId>com.vaadin</groupId>
+                <artifactId>vaadin-radio-button-flow</artifactId>
+                <version>${vaadin.version}</version>
```

Comment:
The version should not be added as they should come from the managedDependencies in the parent test module.
Anything missing should be added there as managed.

Reply:
hmm, isn't it what I am doing in this PR? I mean this is the parent test module, I'm just adding the components as managed dependencies here. 

The dependencies are added in the fusion module in the `pom.xml` file in the `test-spring-security-fusion` module

Reply:
True. The test module naming confuses. sorry.